treeherder/perf/models.py

```diff
@@ -96,16 +96,16 @@ class PerformanceDatum(models.Model):
                              on_delete=models.SET_NULL)
     push = models.ForeignKey(Push)
 
-    ds_job_id = models.PositiveIntegerField(db_column="ds_job_id")
+    # the following properties are obsolete and should be removed at some
```

Comment:
Would you create a follow-up bug to do this removal, if there isn't one already?  :)